srcpkgs/wire-desktop/files/wire-desktop.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Name=Wire
+Comment=The most secure collaboration platform.
+Exec=wire-desktop %U
+Terminal=false
+Type=Application
+Icon=wire-desktop
+StartupWMClass=Wire
+Categories=Network;
+GenericName=Secure messenger
+Keywords=chat;encrypt;e2e;messenger;videocall
+MimeType=x-scheme-handler/wire
+Version=1.1
```

Comment:
It's an open source project; please have this upstream instead of vendoring it in.

Reply:
I don't understand what you mean? Wire isn't creating a desktop file by itself.

Reply:
Yes, but it's their responsibility. Make a PR to them, don't create this file here.

Reply:
https://github.com/wireapp/wire-desktop/issues/4920
Wrote a issue there now.